aldryn_forms/cms_plugins.py

```diff
@@ -172,6 +173,22 @@ class Field(FormElement):
     def get_field_name(self, instance):
         return u'aldryn-forms-field-%d' % (instance.pk,)
 
+    def serialize_value(self, instance, value):
+        if isinstance(value, query.QuerySet):
+            value = ', '.join(map(unicode, value))
```

Comment:
not sure if doing this could cause issues with non ascii characters, maybe use ```u''.join()``` ?